crates/core/src/db/datastore/traits.rs

```diff
@@ -90,7 +90,7 @@ pub trait TxDatastore: DataRow + Tx {
         ctx: &'a ExecutionContext,
         tx: &'a Self::Tx,
         table_id: TableId,
-        cols: NonEmpty<ColId>,
+        cols: impl Into<NonEmpty<ColId>>,
```

Comment:
What's the reason for this change? Seems unrelated, no?

Reply:
This is to keep them consistent with fn `iter_by_col_range_mut_tx()` in `MutTxDataStore`.